content/en/logs/workspaces/_index.md

```diff
@@ -261,3 +261,4 @@ Finally, a treemap visualization cell is created with the `transaction_record_wi
 [2]: https://app.datadoghq.com/logs
 [3]: /logs/log_configuration/parsing/
 [4]: /logs/explorer/calculated_fields/
+[5]: /ddsql_reference/
```

Comment:
Right now, this redirects to the regular current DDSQL page, but I think if you remove the redirect I highlighted above, it'll also fix this redirect so it'll just go to the landing page instead (which I think will be a better customer experience).